src/weapon/w_044.c

```diff
@@ -1,14 +1,114 @@
 // Weapon ID #44. Used by weapons:
 // Heart Refresh
 #include "weapon_private.h"
-extern u16* g_WeaponCluts[];
-extern s32 g_HandId;
-#include "shared.h"
 #include "w_044_1.h"
 #include "w_044_2.h"
 #define g_Animset w_044_1
 #define g_Animset2 w_044_2
 
+static u16 D_138000_8017A040[] = {
+    COLOR16(0, 0, 0, 0),    COLOR16(31, 0, 0, 1),   COLOR16(24, 16, 4, 1),
+    COLOR16(31, 24, 4, 1),  COLOR16(13, 9, 3, 1),   COLOR16(20, 13, 7, 1),
+    COLOR16(24, 18, 12, 1), COLOR16(25, 23, 20, 1), COLOR16(4, 5, 21, 1),
+    COLOR16(7, 9, 31, 1),   COLOR16(4, 6, 4, 1),    COLOR16(8, 9, 8, 1),
+    COLOR16(13, 13, 12, 1), COLOR16(16, 18, 18, 1), COLOR16(21, 22, 22, 1),
+    COLOR16(24, 27, 30, 1), COLOR16(0, 0, 0, 0),    COLOR16(2, 0, 0, 1),
```

Comment:
Open to feedback on how to best format this. If we go hex it'll only go up to `0x1F`:

```c
    COLOR16(0x00, 0x00, 0x00, 0), COLOR16(0x1F, 0x00, 0x00, 1),
    COLOR16(0x18, 0x10, 0x04, 1), COLOR16(0x1F, 0x18, 0x04, 1),
    COLOR16(0x0D, 0x09, 0x03, 1), COLOR16(0x14, 0x0D, 0x07, 1),
    COLOR16(0x18, 0x12, 0x0C, 1), COLOR16(0x19, 0x17, 0x14, 1),
    COLOR16(0x04, 0x05, 0x15, 1), COLOR16(0x07, 0x09, 0x1F, 1),
    COLOR16(0x04, 0x06, 0x04, 1), COLOR16(0x08, 0x09, 0x08, 1),
 ```

We can't include a leading zero for alignment because it'll get interpreted as octal:

```c
    COLOR16(00, 00, 00, 0), COLOR16(31, 00, 00, 1), COLOR16(24, 16, 04, 1),
    COLOR16(31, 24, 04, 1), COLOR16(13, 09, 03, 1), COLOR16(20, 13, 07, 1),
    COLOR16(24, 18, 12, 1), COLOR16(25, 23, 20, 1), COLOR16(04, 05, 21, 1),
    COLOR16(07, 09, 31, 1), COLOR16(04, 06, 04, 1), COLOR16(08, 09, 08, 1),
    COLOR16(13, 13, 12, 1), COLOR16(16, 18, 18, 1), COLOR16(21, 22, 22, 1),
    COLOR16(24, 27, 30, 1), COLOR16(00, 00, 00, 0), COLOR16(02, 00, 00, 1),
```

Reply:
...or we can use a space, if we're ok with turning off or modifying clang format for each section:

```c
COLOR16( 0,  0,  0, 0), COLOR16(31,  0,  0, 1), COLOR16(24, 16,  4, 1),
COLOR16(31, 24,  4, 1), COLOR16(13,  9,  3, 1), COLOR16(20, 13,  7, 1),
COLOR16(24, 18, 12, 1), COLOR16(25, 23, 20, 1), COLOR16( 4,  5, 21, 1),
COLOR16( 7,  9, 31, 1), COLOR16( 4,  6,  4, 1), COLOR16( 8,  9,  8, 1),
COLOR16(13, 13, 12, 1), COLOR16(16, 18, 18, 1), COLOR16(21, 22, 22, 1),
COLOR16(24, 27, 30, 1), COLOR16( 0,  0,  0, 0), COLOR16( 2,  0,  0, 1),
```

Reply:
Since the point of the macro is making it human-readable, I think decimal makes the most sense. I don't mind the alignment but if you want it aligned I think turning off clang-format is reasonable